packages/tools/pie-icons-webc/build.js

```diff
@@ -38,8 +38,7 @@ const componentSelector = '${kebabCase(name)}';
 export class ${name} extends LitElement implements IconProps {
     // The following styles make sure that the icon will be sized correctly
     static styles = css\`
-        :host-context(pie-icon-button) svg,
-        :host-context(pie-button) svg {
+        :host(.has-svg-element) svg {
```

Comment:
Not super keen on this name so happy to update it.

Reply:
Have you verified this works as expected on IOS Safari 14?

Reply:
Host is pretty well supported (back to v10 in Safari) so shouldn't be any issues from browser side, but worth a check too

Reply:
Good point ill check ios as i just checked the browser only.